README.md

```diff
@@ -18,28 +15,23 @@ Are you a Developer using the OCI SDK? If so, please fill out our survey to help
 
 
 ## Installing
-OCI Go SDK supports both `go get` and Go Modules installing.
-
 If you want to install the SDK under $GOPATH, you can use `go get` to retrieve the SDK:
 ```
 go get -u github.com/oracle/oci-go-sdk
 ```
-Alternatively, you can also clone the repo from [Github](https://github.com/oracle/oci-go-sdk).
```

Comment:
If user doesn't use go get command, git clone is also an option.

Reply:
I suspect anyone who would use this code w/o using `go get` would be aware of `git clone`... unlike some other languages, `git` is an integral part of the `go` language ecosystem, so it's safe to assume users are aware... no need to highlight it.